src/core/useCases/feed/feedItem.spec.ts

```diff
@@ -389,7 +392,8 @@ describe('Feed Item', () => {
   it(`
     Given feed has no cached items
       And has selected item uuid
-    When item uuid is a city id
+    When feed has been initialized
+      And item uuid is a city id
```

Comment:
Je pense que faudrait décrire comme tu as fait sur les autres tests avec l'action `set` qui permet d'indiquer qu'on fait une action, et que c'est pas juste une valeur d'origine sur le setup

```suggestion
      And item uuid is set as a city id
```